src/api-service/__app__/onefuzzlib/tasks/main.py

```diff
@@ -153,6 +153,27 @@ def get_by_task_id(cls, task_id: UUID) -> Union[Error, "Task"]:
         task = tasks[0]
         return task
 
+    @classmethod
+    def get_tasks_by_pool_name(
+        cls, pool_name: str
+    ) -> Optional[Union[Error, List["Task"]]]:
+        tasks = cls.search()
+        if not tasks:
+            return Error(code=ErrorCode.INVALID_REQUEST, errors=["unable to find task"])
+
+        pool_tasks = []
+
+        for task in tasks:
+            if not task.config.pool:
```

Comment:
please use `task.get_pool()` to determine the pool used by the task.

Our pre-release API specified pools via task.config.vm.sku/task.config.vm.image.  We need to support tasks from the old configuration method until we can get all of our internal users fully upgraded.

Reply:
(the same change needs to occur in the pool resizing function)